css/application-styles.css

```diff
@@ -0,0 +1,230 @@
+.main-nav {
+  position: static;
+}
+
+.paw-logo {
+  display: inline-block;
+}
+
+/* INPUTS AND LABELS STYLES START */
+input {
+  width:460px;
+  height: 64px;
+  border: 1px solid #D2D2D2;
+  border-radius: 3px;
+  padding: 0;
+  margin-bottom: 20px;
+}
+
+label {
+  font-family: Arial;
+  font-size: 36px;
+  font-weight: 400;
+  line-height: 41px;
+  text-align: left;
+}
+
+/* INPUTS AND LABELS STYLES END */
+
+.grid-container {
+  height: 1012px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.form-applicaion {
+  background: #FFC0CB;
+  border: 1px solid #EDDEDE;
+  border-radius: 12px; 
+  padding: 60px 0 100px 70px;   
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: auto;
+  grid-template-areas: 'title title'
+                        'labelfirst labelast'
+                        'firstName lastname'
+                        'labelemail labeljobrole'
+                        'email jobrole'
+                        'labeladdress .'
+                        'address address'
+                        'cvlabel .'
+                        'cv . '
+                        'sendForm sendForm';
+  height: 80%;
+}
+
+.form-application-title {
```

Comment:
please check the title positioning, in figma it's located higher

Reply:
Thank you, for the alert i think it is fixed now.